src/Overlay/Modal/Modal.tsx

```diff
@@ -35,18 +42,16 @@ export const Modal = forwardRef<HTMLDialogElement, ModalProps>(( props) => {
 
 
     const handleCloseModal = () => {
-      if (onClose) {
-        onClose();
-      }
+      if (onClose) onClose()
       setModalOpen(false);
     };
 
   return (
       <Styled.Dialog {...props} ref={modalRef} onClick={(e) => closeIfClickOutside(e)} className="modal">
-          {hideCancelButton ? null : <Styled.Close icon="close" variant="text" autoFocus onClick={handleCloseModal} /> }
-          {header && <Styled.Header>{header}</Styled.Header>}
-          {children && <Styled.Body>{children}</Styled.Body>}
-          <Styled.Footer>{footer ? footer : <Button {...closeProps} variant="text" onClick={handleCloseModal} />}</Styled.Footer>
+          {hideCancelButton ? null : <Styled.Close className={'cancelBtn'+ ' ' + classNames.cancelBtn} icon="close" variant="text" autoFocus onClick={handleCloseModal} /> }
```

Comment:
I think the default className can just be 'cancel' we know it's a button because it is <button>? Thoughts? Same goes for `closeBtn` -> `close`